proposals/0001-lhls.md

```diff
@@ -0,0 +1,313 @@
+
+Aside from `DISCONTINUITY`, It is not certain which tags need a prefetch prefix to ensure backwards compatibility with players that do not support this proposal. For example, we do not know how Safari's native player will handle a rogue `EXT-X-PROGRAM-DATE-TIME` tag. When test manifest are acquired, implementers must ensure that no unexpected behavior is introduced. 
+
+## What related issues do you consider out of scope for this RFC that could be addressed in the future independently of the solution that comes out of this RFC?
+
+- Alternative connection protocols (WebRTC, Websockets, etc.)
+- Manifestless mode
+
+# Examples
+[examples]: #examples
+
+The example below is a Media Playlist listing two available segments and two prefetch segments.
+
+`URL: https://foo.com/bar.m3u8`
+```
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:2
+#EXT-X-MEDIA-SEQUENCE: 0
+#EXT-X-DISCONTINUITY-SEQUENCE: 0
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:06.531Z
+#EXTINF:2.000
+https://foo.com/bar/0.ts
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:08.531Z
+#EXTINF:2.000
+https://foo.com/bar/1.ts
+
+#EXT-X-PREFETCH:https://foo.com/bar/2.ts
+#EXT-X-PREFETCH:https://foo.com/bar/3.ts
+```
+
+Segments 2 and 3 have a Media Sequence Number of 2 and 3, respectively. The next refresh of the Media Playlist transforms the oldest prefetch segment, and appends a new one:
+
+```
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:2
+#EXT-X-MEDIA-SEQUENCE: 1
+#EXT-X-DISCONTINUITY-SEQUENCE: 0
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:08.531Z
+#EXTINF:2.000
+https://foo.com/bar/1.ts
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:10.531Z
+#EXTINF:2.000
+https://foo.com/bar/2.ts
+
+#EXT-X-PREFETCH:https://foo.com/bar/3.ts
+#EXT-X-PREFETCH:https://foo.com/bar/4.ts
+```
+
+Discontinuities:
+
+```
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:2
+#EXT-X-MEDIA-SEQUENCE: 0
+#EXT-X-DISCONTINUITY-SEQUENCE: 0
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:06.531Z
+#EXTINF:2.000
+https://foo.com/bar/0.ts
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:08.531Z
+#EXTINF:2.000
+https://foo.com/bar/1.ts
+
+#EXT-X-PREFETCH-DISCONTINUITY
+#EXT-X-PREFETCH:https://foo.com/bar/5.ts
+#EXT-X-PREFETCH:https://foo.com/bar/6.ts
+```
+
+In this case `5.ts` and `6.ts` have a Discontinuity Sequence Number of 1. On refresh:
+
+```
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-TARGETDURATION:2
+#EXT-X-MEDIA-SEQUENCE: 1
+#EXT-X-DISCONTINUITY-SEQUENCE: 0
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T20:59:08.531Z
+#EXTINF:2.000
+https://foo.com/bar/1.ts
+#EXT-X-DISCONTINUITY
+#EXT-X-PROGRAM-DATE-TIME:2018-09-05T21:59:10.531Z
+#EXTINF:2.000
+https://foo.com/bar/5.ts
+
+#EXT-X-PREFETCH:https://foo.com/bar/6.ts
+#EXT-X-PREFETCH:https://foo.com/bar/7.ts
+```
+
+`5.ts, 6.ts, and 7.ts` all have the Discontinuity Sequence Number of 1. Note how the `PREFETCH-DISCONTINUITY` transformed to the conventional `EXT-X-DISCONTINUITY` tag, and how that tag still applies to prefetch segments.
```

Comment:
A new `#EXT-X-PROGRAM-DATE-TIME` tag was also introduced. I cannot find this behavior described in earlier sections.

Reply:
It was in an old draft, I'll add it back